ethereum-testnets.asciidoc

```diff
@@ -1,34 +1,49 @@
 [[testnets]]
-== Testnet Overview
+== Ethereum Test Networks (Testnets)
+
 === What is a testnet?
 
 A test network (testnet for short) is used to simulate the behavior of the main Ethereum network. There are some publicly available test networks that are simply alternative Ethereum blockchains. The currency on these networks is worthless, but they are still useful since  functionality of contracts and protocol changes can be tested without disrupting the main Ethereum network or using real money. When any major change to the Ethereum protocol is about to be included in the main network, its tests are mostly done on these test networks. These test networks are also used by a large number of developers for testing applications before deploying them onto the main network.
 
 === Using Testnets
 
-You can either connect to publicly available test networks or spawn a development network of your own. First, let's use a public testnet for easier setup. To use a public testnet requires some testnet ethers and a connection to that network. For testnet ethers, faucets are used. To connect to testnets, we will discuss how to do so in various popular clients.
+You can either connect to publicly available test networks or spawn a private test network of your own. First, let's use a public testnet for easier setup. To use a public testnet requires some testnet ether and a connection to that network. For testnet ether, "faucets" are used, which distribute test ether slowly, "dripping" out small amount to anyone who asks. To connect to a testnet, you need an Ethereum client, either a full client such as geth, or a lightweight client such as MetaMask.
```

Comment:
I agree with @rphmeier. For me it is unnatural to call lightweight client to MetaMask, although honestly, I still do not know the proper term. I think we need to think a bit about this point.

Reply:
The closest analogue to an SPV client in Ethereum is an actual light client, which verifies the header chain and fetches state data (pinned to the merkle root) only as necessary to serve user requests. This is a terminology that has been used since well before the launch of the Ethereum mainnet.